cmd/tnf/generate/catalog/catalog.go

```diff
@@ -116,6 +121,85 @@ func emitTextFromFile(filename string) error {
 	return nil
 }
 
+// it turns a list of identifiers
+// { url, version  }
+// and takes urls like http://test-network-function.com/testcases/SuiteName/TestName
+// to build a more structured catalogue
+// {
+//     suiteNameA: [testName [identifiers], testName2 [identifiers] ]
+//     suiteNameB: [testName3 [identifiers], testName4 [identifiers] ]
+// }
+func createPrintableCatalogFromIdentifiers(keys []claim.Identifier) map[string][]catalogElement {
+	catalog := make(map[string][]catalogElement)
+	// we need the list of suite's names
+	var element catalogElement
```

Comment:
Don't need to declare `element` here, just append to the map like my below suggestion.

Reply:
done it